pkg/csi/cinder/openstack/openstack.go

```diff
@@ -61,6 +64,7 @@ type Config struct {
 		DomainId   string `gcfg:"domain-id"`
 		DomainName string `gcfg:"domain-name"`
 		Region     string
+		CAFile     string `gcfg:"ca-file"`
```

Comment:
just a reminder, later on we can add a section in doc about the CAFile setting..

Reply:
Good idea @jichenjc